packages/terra-application-name/src/ApplicationMenuName.scss

```diff
@@ -2,7 +2,7 @@
 :local {
   .application-menu-name {
     align-items: var(--terra-application-menu-name-align-items, center);
-    color: var(--terra-application-menu-name-color, #fff);
+    color: var(--terra-application-menu-name-color, hsl(0, 0%, 0%));
```

Comment:
Should we use #000 instead of hsl(0, 0%, 0%) to be consistent with other component's styles?